### DISCUSSION TITLE
Data Encryption

### DISCUSSION THREAD

**Discussion by olimart** [AllYourBot/hostedgpt#233] [General]
Please see https://edgeguides.rubyonrails.org/active_record_encryption.html

I think we should encrypt:
- user: first & last name
- person: email

However, it implies dealing with Rails credentials/env. variable to share keys without breaking installed apps.

**Comment by keithschacht** [comment]
Hi @olimart, Stephen is picking up the encryption task with this PR:
https://github.com/allyourbot/hostedgpt/pull/274

I chimed in with some comments and questions but you are actually more familiar at with this than I am. Could you take a look?

**Comment by keithschacht** [comment]
This actually got implemented a couple months back, fyi.
